docs/source/backends/impala.rst

```diff
@@ -867,16 +1011,15 @@ must take place each time you load your Ibis session.
 
 The object ``fuzzy_equals`` is callable and works with Ibis expressions:
 
-.. code-block:: ipython
+.. code-block:: python
 
-   In [35]: db = c.database('ibis_testing')
+   >>> db = c.database('ibis_testing')
 
-   In [36]: t = db.functional_alltypes
+   >>> t = db.functional_alltypes
 
-   In [37]: expr = fuzzy_equals(t.float_col, t.double_col / 10)
+   >>> expr = fuzzy_equals(t.float_col, t.double_col / 10)
 
-   In [38]: expr.execute()[:10]
-   Out[38]:
+   >>> expr.execute()[:10]
```

Comment:
should these use the ipython block style? (e.g. what you have at the left)?

Reply:
They surely can. I personally don't find that there is a big difference for the user between the `In [x]` and the `>>>` (let me know if you think there is). So, I just tried to keep it consistent in the whole page (as it was). The difference from changing it from one to the other, is that I find the `>>>` much easier to maintain, since it's easier to write, and also adding a block or a new line at the begging of the page would imply to renumber all the blocks in the document. And I'm going to move the impala specific stuff from the tutorial to this page immediately after this PR, so the ipython syntax is just going to make my life more difficult, for IMHO no gain.

Is there something am I missing, or do you think the ipython syntax is so much clear to make it worth the hassle?

Reply:
yeah if the *entire* page is consistent great. since we have mostly generated example do prefer the ipython style as that matches (but i here you about keeping it up).

either way really.

Reply:
Yes, the entire page is using regular python code blocks, there are no ipython blocks, neither auto-generated or manual

Reply:
kk fine by me